test/parallel/test-fs-read-file-sync-hostname.js

```diff
@@ -1,10 +1,14 @@
 'use strict';
-require('../common');
-var assert = require('assert');
-var fs = require('fs');
+const common = require('../common');
+const assert = require('assert');
+const fs = require('fs');
 
-// test reading from hostname
-if (process.platform === 'linux2') {
-  var hostname = fs.readFileSync('/proc/sys/kernel/hostname');
-  assert.ok(hostname.length > 0);
+if (!common.isLinux) {
```

Comment:
I thought `linux2` was a python thing and was misspelled. Are there any linuxes where `process.platform` is set to `linux2`? From the conversation in the mailing list where the patch that introduced the test was [discussed](https://groups.google.com/forum/#!topic/nodejs-dev/rxZ_RoH1Gn0), it seems that the problem was that in using `stat` on a file in the `/proc` directory caused issues and `readFileSync` should not use any `stat` call in its implementation. Then, looking the `stat` manual found this:

> For most files under the /proc directory, stat() does not return the file size in the st_size field; instead the field is returned with the value 0.

So if there are `linux2` systems out there I think it would probably make sense testing in both `linux` and `linux2`. If someone could shed some light on this, it would be great.